tmt/base.py

```diff
@@ -934,6 +929,64 @@ def stories(
                 keys=keys, names=names, whole=whole)],
             filters, conditions)
 
+    @staticmethod
+    def init(context, path, template, force, **kwargs):
+        _init_templates = listed(tmt.templates.INIT_TEMPLATE_CHOICES,
+                                 join='or')
+        path = os.path.realpath(path)
+        dry = Tree._opt('dry')
+
+        # Check for existing tree
+        try:
+            tree = tmt.Tree(path)
+            # Are we creating a new tree under the existing one?
+            if path == tree.root:
+                echo("Tree '{}' already exists.".format(tree.root))
+            else:
+                tree = None
+        except tmt.utils.GeneralError:
+            tree = None
+
+        # Create a new tree
+        if tree is None and dry:
+            tree = tmt.Tree(path)
+            echo("Tree '{}' would be initialized.".format(tree.root))
+        elif tree is None and not dry:
+            try:
+                fmf.Tree.init(path)
+                tree = tmt.Tree(path)
+            except fmf.utils.GeneralError as error:
+                raise tmt.utils.GeneralError(
+                    "Failed to initialize tree in '{}': {}".format(
+                        path, error))
+            echo("Tree '{}' initialized.".format(tree.root))
+
+        # Populate the tree with example objects if requested
+        if template == 'empty':
+            non_empty_choices = [c for c in tmt.templates.INIT_TEMPLATE_CHOICES
+                                 if c != 'empty']
+            echo(
+                "To populate it with example content, use --template with "
+                "{}.".format(listed(non_empty_choices, join='or')))
+        elif dry:
+            echo("Would apply the template '{}'.".format(template,
```

Comment:
This is just a message, nothing happens here, let's remove this `elif` part completely.